packages/matchbox/src/components/Table/styles.js

```diff
@@ -0,0 +1,28 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const table = () => `
+  position: relative;
+  text-align: left;
+  width: 100%;
+  padding: 0;
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+`;
+
+export const headerCell = () => `
+  font-size: ${tokens.fontSize_200};
+  line-height: ${tokens.lineHeight_200};
+  font-weight: ${tokens.fontWeight_semibold};
+`;
+
+export const cell = () => `
+  word-break: break-all;
+`;
+
+export const row = () => `
+  background: ${tokens.color_white};
+  &:nth-of-type(even) {
+    background: ${tokens.color_gray_100};
+  }
+`;
```

Comment:
I'm not sure where to put this, or if we can even do this, but the mocks have a line under the thead header row.

Reply:
Wondering if we need a new component, what do you think?